vars/documentationChecker.groovy

```diff
@@ -0,0 +1,51 @@
+
+
+// Add the following code to any Jenkinsfile.
+// It will return with success if there are no changes in the specified docs dir
+// and Will return with success if run on a non Pull Request branch
+
+// documentationChecker("docs")
+
+// docsDir ~~ the subdirectory within your repo that the documentation resides
+def documentationChecker(docsDir) {
+  if (env.CHANGE_ID != null) {
```

Comment:
The hope was that this would stop the code being triggered when its not a PR - the docs imply that its set when there's a "some kind of change request"

looking at the output of `env` in https://ci.qa.aws.dckr.io/job/Documentation%20pipeline/job/docs.docker.com/job/master/lastSuccessfulBuild/consoleFull

it looks to me like CHANGE_ID is not set when its run on a branch, not a pull request.

/me hopes it'll work :)